packages/contracts/contracts/governance/SecurityCouncilToken.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+import { Semver } from "../universal/Semver.sol";
+import "../universal/KromaSoulBoundERC721.sol";
+
+/**
+ * @custom:proxied
+ * @title SecurityCouncilToken
+ * @notice The SecurityCouncilToken is a basic token based KromaSoulBoundERC721.
+ */
+contract SecurityCouncilToken is
+    Initializable,
+    KromaSoulBoundERC721,
+    Semver {
+    /**
+     * @custom:semver 0.1.0
+     */
+    constructor()
+        Semver(0, 1, 0)
+    {}
+
+    /**
+     * @notice Initializer.
+     *
+     * @param _owner           Owner of this token contract.
+     */
+    function initialize(address _owner) initializer public {
+        __KromaSoulBoundERC721_init("KromaSecurityCouncil", "KSC", _owner);
+    }
+
+    function _baseURI() internal pure override returns (string memory) {
+        return "";
```

Comment:
If this would be base URI for security council SBT, please mark TODO for future use.